src/Microsoft.OpenApi/Interfaces/IOpenApiReferenceable.cs

```diff
@@ -25,7 +26,7 @@ public interface IOpenApiReferenceable : IOpenApiSerializable
         /// <summary>
         /// Serialize to OpenAPI V3 document without using reference.
         /// </summary>
-        void SerializeAsV3WithoutReference(IOpenApiWriter writer);
+        void SerializeAsV3WithoutReference(IOpenApiWriter writer, OpenApiSpecVersion version, SerializeDelegate callback);
```

Comment:
shouldn't this get another method SerializeAsV31WithoutReference to be consistent?

In which case we wouldn't need the version or the callback arguments anymore and that'd become implementation details

Reply:
I can add that too.